ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/disconnected/provision/cucushift-installer-rehearse-vsphere-ipi-disconnected-provision-chain.yaml

```diff
@@ -2,7 +2,9 @@ chain:
   as: cucushift-installer-rehearse-vsphere-ipi-disconnected-provision
   steps:
   - chain: ipi-conf-vsphere
-  - ref: ipi-conf-vsphere-mirror-image
+  - ref: ipi-conf-vsphere-mirror-registry
+  - ref: mirror-images-by-oc-adm
```

Comment:
For vsphere disconnected install, does need `ipi-install-rbac` before `ref: mirror-images-by-oc-adm` ?